app/views/tracks/index.haml

```diff
@@ -9,6 +9,12 @@
 = render :partial => "application/subnav", :locals => {:season => current_season}
 
 #tracks
+  = form_tag(tracks_path, method: :get) do
+    .form-group
+      = text_field_tag :query, params[:query], class: 'form-control', placeholder: t("rva.cars.shared.placeholder")
```

Comment:
I feel like we should move `Search...` to the misc strings. It's generic enough, and now it's needed somewhere that isn't related to the Cars module.

Once you do, make sure to look for all usages of `rva.cars.shared.placeholder` and replace them accordingly with the new string key.